src/components/userProfile/RequestPage.js

```diff
@@ -79,41 +79,43 @@ class RequestPage extends Component {
       <ScrollView contentContainerStyle={styles.content}>
         <Text style={styles.title}>What would you like to ask first?</Text>
         <Text style={styles.subTitle}>Subjects</Text>
-        <MenuContext ref="MenuContext">
-          <View style={styles.dropdownContent}>
-            <Menu style={styles.dropdown}
-              onSelect={(value) => this.setState({ selection: value })}>
-              <MenuTrigger>
-                <Text>{this.state.selection}</Text>
-              </MenuTrigger>
-              <MenuOptions optionsContainerStyle={styles.dropdownOptions}
-                renderOptionsContainer={(options) =>
-                  <ScrollView>{options}</ScrollView>}>
-                <MenuOption value='Study abroad'>
-                  <Text>Study abroad</Text>
-                </MenuOption>
-                <MenuOption value='Get a new job'>
-                  <Text>Get a new job</Text>
-                </MenuOption>
-                <MenuOption value='Portfolio & resume'>
-                  <Text>Portfolio & resume</Text>
-                </MenuOption>
-                <MenuOption value='Start up'>
-                  <Text>Start up</Text>
-                </MenuOption>
-                <MenuOption value='Career change'>
-                  <Text>Career change</Text>
-                </MenuOption>
-                <MenuOption value='Networking'>
-                  <Text>Networking</Text>
-                </MenuOption>
-                <MenuOption value='Soft skills'>
-                  <Text>Soft skills</Text>
-                </MenuOption>
-              </MenuOptions>
-            </Menu>
-          </View>
-        </MenuContext>
+        <View style={{ zIndex: 500 }}>
+          <MenuContext ref="MenuContext">
+            <View style={styles.dropdownContent}>
+              <Menu style={styles.dropdown}
+                onSelect={(value) => this.setState({ selection: value })}>
+                <MenuTrigger>
+                  <Text>{this.state.selection}</Text>
+                </MenuTrigger>
+                <MenuOptions optionsContainerStyle={styles.dropdownOptions}
+                  renderOptionsContainer={(options) =>
+                    <ScrollView>{options}</ScrollView>}>
+                  <MenuOption value='Study abroad'>
```

Comment:
How about making a resource file for object that contains the name of menus?
(like 'Study abroad', 'Get a new job', ... )
That would be better for maintaining.